.github/integration/sda-s3-integration.yml

```diff
@@ -317,13 +317,78 @@ services:
       - ./sda/config.yaml:/config.yaml
       - shared:/shared
 
+  cega-nss:
+    container_name: cega-nss
+    depends_on:
+      credentials:
+        condition: service_completed_successfully
+    command:
+      [
+        "python",
+        "/cega/users.py",
+        "0.0.0.0",
+        "8443",
+        "/shared/users.json"
+      ]
+    environment:
+      - CEGA_USERS_PASSWORD=test
+      - CEGA_USERS_USER=test
+    image: "egarchive/lega-base:release.v0.2.0"
+    ports:
+      - "8443:8443"
+    volumes:
+      - ./sda/users.py:/cega/users.py
+      - shared:/shared
+
+  auth-cega:
+    command: [ sda-auth ]
+    container_name: auth-cega
+    depends_on:
+      cega-nss:
+        condition: service_started
+    environment:
+      - AUTH_RESIGNJWT=true
+      - AUTH_CEGA_ID=test
+      - AUTH_CEGA_SECRET=test
+    image: ghcr.io/neicnordic/sensitive-data-archive:PR${PR_NUMBER}
+    ports:
+      - "8888:8080"
+    restart: always
+    volumes:
+      - ./sda/config.yaml:/config.yaml
+      - shared:/shared
+
+  auth-oidc:
+    command: [ sda-auth ]
+    container_name: auth-oidc
+    depends_on:
+      oidc:
+        condition: service_healthy
+    environment:
+      - AUTH_RESIGNJWT=false
+      - OIDC_ID=XC56EL11xx
+      - OIDC_SECRET=wHPVQaYXmdDHg
+    image: ghcr.io/neicnordic/sensitive-data-archive:PR${PR_NUMBER}
```

Comment:
Why have two services? Is it just to test that they can deploy with separate logins? Seems a bit too cluttered.

Reply:
Because the auth tests are fubar as pointed out in the PR text, but this can be solved later since there are other service that rely on the NSS/OIDC endpoints.

Reply:
Maybe I was unclear, I meant why not configure one `auth:` with both cega and oidc. But in any case, testing can be revisited on a separate issue.